workers/unity/Packages/io.improbable.gdk.ragdolls/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "io.improbable.gdk.ragdolls",
+  "displayName": "SpatialOS GDK Ragdolls",
+  "version": "0.2.3",
```

Comment:
Is this rebased on our 0.2.4 release? since this is old?

Reply:
My guess is no based on these comments 😅

Reply:
Oof, enjoy that rebase